### PR TITLE
Update admin-site to support json schema

### PR DESCRIPTION
**Pull request by norbye**
For the main backend changes, see #1178.

Updates the admin-views to support viewing applications, download CSVs and edit the JSON schema.

The backend + the frontend structure supports adding fields, reordering fields, etc. but this is not implemented in the GUI yet. Would rather do this in a later PR.

The functionality to change the top questions (info and phone number in the screenshot in #1179) is not yet implemented, but will match the visual preview for group-leaders.

### Admin page for group-leaders and recruiters
image

image

### Admin page for admission-admins
image

Resolves ABA-515, ABA-516

### CODE REVIEW

File: frontend/src/utils/methods.ts
Comment (PeterJFB) [inline frontend/src/utils/methods.ts:33]: Looks unused? Or will it be utilized later?

File: frontend/src/utils/methods.ts
Comment (PeterJFB) [inline frontend/src/utils/methods.ts:52]: Rename to `uuidv4Mock` or something similar to clarify its use-case:)

File: frontend/src/containers/UserApplication/index.tsx
Comment (PeterJFB) [inline frontend/src/containers/UserApplication/index.tsx:62]: Missing whitespace? :eyes:

File: frontend/src/components/JsonFieldEditor/TextArea.tsx
Comment (PeterJFB) [inline frontend/src/components/JsonFieldEditor/TextArea.tsx:24]: Looks like `JsonFieldTextArea`s placeholder-field is supposed to be used here :thinking:

**Review by PeterJFB — APPROVED** [review]
First look-through looks nice :100: 
Left a few comments but nothing critical considering it being an mvp

**Comment by norbye** [comment]
Converted to draft to ensure it is not merged before 2023 admissions, will merge some time after that.

File: frontend/src/utils/methods.ts
Comment (norbye) [reply frontend/src/utils/methods.ts:33]: Seems I added it to the wrong branch - it will be used when merged with the others:)

**Pull request merged**
